pkg/kiosk/anonymous_login.go

```diff
@@ -45,7 +46,15 @@ func GrafanaKioskAnonymous(cfg *Config, messages chan string) {
 	/*
 		Launch chrome and look for main-view element
 	*/
+	headers := make(map[string]interface{})
+	if len(cfg.BasicAuth.Username) != 0 && len(cfg.BasicAuth.Password) != 0 {
```

Comment:
why would anonymous login require authentication?

Reply:
the anonymous login is for Grafana, meaning that the grafana dashboard (app) itself doesn't require authentication (user account) but it still might be hidden behind a reverse proxy that requires HTTP-Basic auth.
So I see them as two separate authentication requirements.